### PR TITLE
Update repo's name and rbac generation template, tiflow cluster example

### PR DESCRIPTION
**Pull request by lichunzhu**
reference: https://github.com/pingcap/tiflow/pull/6435/files#r934102913

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,5 @@ testbin/*
 *.swp
 *.swo
 *~
+
+.DS_Store
```
Comment (amyangfei) [inline .gitignore]: Can remove this file too.

Comment (lichunzhu) [reply .gitignore]: which file?

Comment (amyangfei) [reply .gitignore]: I mean the `.DS_Store` file

File: examples/_v1alpha1_tiflowcluster.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ spec:
       etcd-endpoints = ["etcd:2379"]
       [frame-metastore-conf]
```
Comment (amyangfei) [inline examples/_v1alpha1_tiflowcluster.yaml]: s/frame/framework

File: .gitignore (hunk 1 above)
Comment (lichunzhu) [reply .gitignore]: It's mac's system file, tiflow also has this in gitignore.
https://github.com/pingcap/tiflow/blob/master/.gitignore#L29

Comment (amyangfei) [reply .gitignore]: Yep, I mean to keep `.DS_Store` in `.gitignore` file, and also remove the file `.DS_Store` from repo

**Review by arlottang — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (lichunzhu) [reply .gitignore]: Gotta. I will delete them in the next PR.

**Pull request merged**